application/forms/RotationConfigForm.php

```diff
@@ -245,12 +231,25 @@ public function loadRotation(int $rotationId): self
                 return $handoff;
             };
 
+            /** @var ?Rotation $rotation */
+            $rotation = Rotation::on($this->db)
+                ->filter(Filter::all(
+                    Filter::equal('id', $this->rotationId),
+                    Filter::equal('deleted', 'n')
+                ))
+                ->first();
+            if ($rotation === null) {
+                throw new HttpNotFoundException($this->translate('Rotation not found'));
+            }
```

Comment:
This shouldn't be required. Just update the code to access what `fetchDbValues()` returns, please.